ColorPatterns/Support/AppDelegate.swift

```diff
@@ -14,7 +14,9 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        //  Show lauch screen for 0.5 sec to display logo
+
+        // TODO: Never seen that kind of logic here, but It could be smarter to add subview of launchscreen to your game controller
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Support/AppDelegate.swift#L18"></span>TODOs should be resolved (Never seen that kind of logic …).<br />
<code>todo</code> <code>AppDelegate.swift:18</code></td>
    </tr>
  </tbody>
</table>